.generator/pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "generator"
+version = "0.1.0"
+description = ""
+authors = ["Jiri Kuncar <jiri.kuncar@datadoghq.com>"]
```

Comment:
😉

Reply:
😄 . Should we put some generic name like support@?

Reply:
Probably :)